gap/matrix.gi

```diff
@@ -763,6 +767,12 @@ InstallGlobalFunction( FindKernelLowerLeftPGroup,
     return true;
   end );
 
+# computes a straight line program (SLP) for an element <g> of for a p-group
+# in <ri> (that corresponds to a matrix group consisting of lower triangular
+# matrices.
+# The SLP is constructed by using the elementar matrices to cancel out the
```

Comment:
```suggestion
# The SLP is constructed by using the elementary matrices to cancel out the
```
Though actually that's not correct, the matrices used for reduction are in general not elementary. Perhaps like this:
```suggestion
# The SLP is constructed by using matrices forming a pcgs (polycyclic generating sequence)
# of the group to clean out the entries of the input matrix in the lower left part entry by entry.
```